test/control-plane/services/dual-stack/dualstack_test.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we have these files `test/control-plane/services/dual-stack` to be part of a separate commit?

Reply:
Done.